doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -163,11 +167,9 @@ For a given mode of the ring resonator, it is often useful to know how sensitive
 
 [Pertubation theory for Maxwell equations involving high index-contrast dielectric interfaces](http://math.mit.edu/~stevenj/papers/KottkeFa08.pdf) is reviewed in Chapter 2 of [Photonics Crystals: Molding the Flow of Light, 2nd Edition (2008)](http://ab-initio.mit.edu/book/). The formula (equation 30 on p.19) for the frequency shift $\Delta \omega$ resulting from the displacement of a block of $\varepsilon_1$-material towards $\varepsilon_2$-material by a distance $\Delta h$ (perpendicular to the boundary) is:
 
-<center>
```

Comment:
Is it possible to also apply the `<p align="center">...</p>" HTML construct to center Latex equations?

Reply:
I'll give it a try later today.

Reply:
These three gave the same display. 
The \$\$ ... \$\$ is supposed to center it, at least from the pure $\LaTeX$ behavior. 


<center>

$$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$

</center>

$$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$

<p align="center">
   $$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$
</p>